docker-neo4j/azuredeploy.json

```diff
@@ -0,0 +1,399 @@
+            },
+            "adminUsername": {
+                "type": "string",
+                "metadata": {
+                    "description": "Username for sample VM"
+                }
+            },
+            "adminPassword": {
+                "type": "securestring",
+                "metadata": {
+                    "description": "User password for sample VM"
+                }
+            },
+            "newStorageAccountName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Stoarge Account Name for VM Disk"
+                }
+            },
+            "storageAccountType": {
+                "type": "string",
+                "allowedValues": [
+                    "Standard_LRS",
+                    "Standard_GRS",
+                    "Standard_RAGRS",
+                    "Premium_LRS"
+                ],
+                "metadata": {
+                    "description": "The type of the Storage Account created"
+                },
+                "defaultValue": "Standard_LRS"
+            },
+            "networkSecurityGroupName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Name of the network security group."
+                },
+                "defaultValue": "myNSG"
+            },
+            "networkInterfaceName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Name of the network security group."
+                },
+                "defaultValue": "myVMNic"
+            },
+            "publicIPAddressName": {
+                "type": "string",
+                "metadata": {
+                    "description": "Name of the public IP address"
+                },
+                "defaultValue": "myPublicIP"
+            }
+        },
+        "variables": {
+            "imagePublisher": "Canonical",
+            "imageOffer": "UbuntuServer",
+            "ubuntuOSVersion": "15.04",    
+            "OSDiskName": "osDisk",
+            "nsgName": "[parameters('networkSecurityGroupName')]",
+            "nicName": "[concat(parameters('vmName'),parameters('networkInterfaceName'))]",
+            "dockerExtensionName": "DockerExtension",
+            "azureVMUtilsName": "vmCustomizationExt",
+            "azureVMUtilsScriptUrl": "https://github.com/Azure/docker-neo4j/raw/master/extras/setup_data_disk.sh",
+            "addressPrefix": "10.0.0.0/16",
+            "subnetName": "Subnet",
+            "subnetPrefix": "10.0.0.0/24",
+            "storageAccountType": "Standard_LRS",
+            "publicIPAddressName": "[concat(parameters('vmName'),parameters('publicIPAddressName'))]",
+            "publicIPAddressType": "Dynamic",
+            "vmStorageAccountContainerName": "vhds",
+            "vmName": "[parameters('vmName')]",
+            "vmSize": "[parameters('vmSize')]",
+            "virtualNetworkName": "MyVNET",
```

Comment:
For this and other variable names it'd be a good idea to prefix the scenario/application of this template (e.g. neo4j) so that when a user lists all VNETs in portal, there aren't 10 VNET's name myVNET :)